### ISSUE TITLE
Form helpers

### ISSUE THREAD

**Issue by sauliusgrigaitis** [necolt/Swifton#7]
- [ ] Investigate http://guides.rubyonrails.org/form_helpers.html and check what most important helpers do we need.
- [ ] Investigate https://github.com/kylef/Stencil#customisation and findout how it can help

**Comment by habibalamin** [comment]
Might want to check out `simple_form` as well. That's the most popular form helper gem for Rails, which enhances the default helpers.

**Comment by sauliusgrigaitis** [comment]
Yes, we need to analyse popular Ruby gems for form helpers https://www.ruby-toolbox.com/categories/rails_form_builders and then figure out what to start with.

**Comment by RyanCCollins** [comment]
The form helper I prefer is formtastic. It's well supported and has a very comprehensive API. It may have too many dependencies, in which case, simple_form is the way to go.  I'm a rails and swift developer and am hoping I can help with this project at some point. I love what you're doing.  I hope this helps. https://rubygems.org/gems/formtastic
